.env.example

```diff
@@ -0,0 +1 @@
+MONGODB_URL="CONNECTION STRING"
```

Comment:
Add the file line endings so we dont get this red signal, red == trigger

Reply:
Fixed